src/nora/components/KabobMenu/KabobMenuContainer.md

```diff
@@ -0,0 +1,39 @@
+```jsx
+import styles from './KabobExample.module.scss'
+import { Body2, Spacer } from '../../../components/index'
+
+const ExampleKabobMenuConsumer = () => {
+  const [isOpen, setIsOpen] = React.useState(false)
+  const [activeOption, setActiveOption] = React.useState(0)
```

Comment:
nit: no need for the 'activeOption' in this example? or did you mean to use it in the example?

Reply:
Yup, thanks Sriram, fixed in:
67433acdc0448c827f8c68749b8c1045f3d106d5